src/components/presentational/Popup/Popup.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { faChevronDown } from '@fortawesome/free-solid-svg-icons/faChevronDown';
+import { FontAwesomeSvgIcon } from 'react-fontawesome-svg-icon';
+
+import './Popup.css';
+
+export interface PopupProps {
+    children?: React.ReactNode;
+    onCollapse: () => void;
+    title?: string;
+    titleIcon?: React.JSX.Element;
+}
+
+export default function (props: PopupProps) {
+
+    return <div className="mdhui-popup">
+        <div className="mdhui-popup-header">
+            <div>
```

Comment:
I would use an Action component here which supports a left/right icon

Reply:
I don't think `Action` is the right component to use here. If we really wanted to take reusability all the way we can probably write an Icon component and it would have the option to have some text next to the icon as well. But this is small enough and simple enough I would prefer to just leave it as is icon with some text next to it doesn't need to be a component.

Reply:
Why not?  This seems to perfectly match action because it has the same layout as action?  Since it has a left/right icon which action supports...and again Action has all the correct styles associated with it etc.